snapshot-labs/sx-monorepo
# fix(ui): set a min textarea height

### Summary



This PR add a minimum height to style textarea, to avoid UI bug like 

![Screenshot 2024-06-25 at 15 52 26](https://github.com/snapshot-labs/sx-monorepo/assets/495709/c4229db3-53be-4982-a85a-a83f9c927cd5)

After fix, the smallest textarea height should be the same as regular input (61px), and looks like a text input.

![Screenshot 2024-06-25 at 15 55 14](https://github.com/snapshot-labs/sx-monorepo/assets/495709/680aa645-71d6-4c10-9c02-bb69f3b5e9b1)

PR also fix an UX issue where clicking on the label was not focusing on the input field

### How to test

1. Go to any form with texarea
2. Resize the textare height to minimum
3. Smallest height should be 61px, same height as `input`
5. Click on the label
6. It should focus on the input
